Metagenomics/slides/index.html

```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+<html lang="en">
+
```

Comment:
will do! it's just the example file I copied while back from template folder, could also just remove it?